sdk/sdkservices/sessions.go

```diff
@@ -13,12 +13,22 @@ type ListSessionsFilter struct {
 	BuildID      sdktypes.BuildID
 	StateType    sdktypes.SessionStateType
 	CountOnly    bool
+
+	PageSize  int32
+	Skip      int32
+	PageToken string
```

Comment:
i think these should be common to all paginated results. put these in a common struct and just reuse it all over.

Reply:
add `TotalCount`